tests/foreman/api/test_remoteexecution.py

```diff
@@ -50,7 +47,6 @@ def test_positive_run_capsule_upgrade_playbook(module_capsule_configured, target
         data={
             'job_template_id': template_id,
```

Comment:
According to https://github.com/theforeman/foreman_ansible/pull/727 you should also change job_template_id from `Capsule Upgrade Playbook` to `Capsule Update Playbook` 
(unless you provide next version maintenance repo in order to upgrade to Y+1 successfully)